config/routes.rb

```diff
@@ -9,6 +9,7 @@
   resources :public_keys
   resource :lti_config
 
+  resources :lti_dynamic_registrations
```

Comment:
Only :index is in the controller, should we restrict this? Or will downstream be adding more?

Reply:
I think we'll at least need :show at some point. I'm inclined to leave it for now. Is there any risk in leaving it?

Reply:
I think we leave it for now. It is likely that we will need additional endpoints in the downstream apps.

Reply:
ok 👍